app/services/email_sender/notify.rb

```diff
@@ -2,33 +2,33 @@
 
 module EmailSender
   class Notify
-    def call(address:, **keyword_args)
-      send_to_notify(address: address, **keyword_args)
-    end
-
-  private
-
-    def send_to_notify(address:, **keyword_args)
+    def call(address:, subject:, body:)
       client.send_email(
         email_address: address,
-        template_id: template_id(keyword_args)
+        template_id: template_id,
+        personalisation: {
```

Comment:
This is much better just going on in here. We're in `Notify` land in this class which is defo where the translation from our world should happen.